packages/frontend/src/presentation/screens/CardSearch/CardSearchFilterLayout.tsx

```diff
@@ -27,40 +102,44 @@ export const CardSearchFilterLayout: React.FC<Props> = (props: Props) => {
-          onValueChange={(itemValue: string): void => setType(itemValue)}
-        >
-          <Picker.Item label="Type" value="" />
-          <Picker.Item label="Basic" value="basic" />
-          <Picker.Item label="Conspiracy" value="conspiracy" />
-          <Picker.Item label="Creature" value="creature" />
-          <Picker.Item label="Enchantment" value="enchantment" />
-          <Picker.Item label="Instant" value="instant" />
-          <Picker.Item label="Land" value="land" />
-          <Picker.Item label="Legendary" value="legendary" />
-          <Picker.Item label="Planeswalker" value="planeswalker" />
-          <Picker.Item label="Sorcery" value="sorcery" />
-          <Picker.Item label="Summon" value="summon" />
-          <Picker.Item label="Tribal" value="tribal" />
-          <Picker.Item label="Vanguard" value="vanguard" />
-        </Picker>
-        <Picker
-          style={styles.picker}
-          selectedValue={format}
-          onValueChange={(itemValue: string): void => setFormat(itemValue)}
-        >
-          <Picker.Item label="Format" value="" />
-          <Picker.Item label="Brawl" value="brawl" />
-          <Picker.Item label="Commander" value="commander" />
-          <Picker.Item label="Legacy" value="legacy" />
-          <Picker.Item label="Modern" value="modern" />
-          <Picker.Item label="Pauper" value="pauper" />
-          <Picker.Item label="Pionner" value="pionner" />
-          <Picker.Item label="Standard" value="standard" />
-          <Picker.Item label="Vintage" value="vintage" />
-        </Picker>
+        <View style={styles.inputGroup}>
+          <View
+            style={{ ...styles.autoCompleteInputContainer, marginRight: 2 }}
+          >
+            <Input
+              value={subtype}
+              autoCapitalize="none"
+              placeholder="Sub(Type)"
+              onChangeText={setSubtype}
+            />
+          </View>
+          <View style={{ ...styles.autoCompleteInputContainer, marginLeft: 2 }}>
+            <Input
+              value={supertype}
+              autoCapitalize="none"
+              placeholder="Super(Type)"
+              onChangeText={setSupertype}
+            />
+          </View>
+        </View>
+        <Text style={styles.regularText}>Color</Text>
+        <View style={styles.manaRow}>
+          <ManaColorCheckbox color="W" isSelected={colors.includes('W')} />
+          <ManaColorCheckbox color="B" isSelected={colors.includes('B')} />
+          <ManaColorCheckbox color="R" isSelected={colors.includes('R')} />
+          <ManaColorCheckbox color="U" isSelected={colors.includes('U')} />
+          <ManaColorCheckbox color="G" isSelected={colors.includes('G')} />
+          <ManaColorCheckbox color="C" isSelected={colors.includes('C')} />
+        </View>
+        <Text style={styles.regularText}>Color Identity</Text>
+        <View style={styles.manaRow}>
+          <ManaIdentityCheckbox color="W" />
```

Comment:
color deveria ser representado como um tipo.
```suggestion
          <ManaIdentityCheckbox color={Colors.WHITE} />
```